client/components/popover/util.js

```diff
@@ -38,42 +38,45 @@ const adjacent = {
 	right: 'bottom',
 };
 
-//let viewport;
+let _viewport = null;
+let _windowEventsRefCount = 0;
 
-let viewport = updateViewport();
+function getViewport() {
```

Comment:
Yes, I'd like to avoid unnecessary dependencies, and `once` wouldn't work here anyway. The `viewport` variable can also be updated by the `onViewportChange` listener. It's initialized lazily on first access, and then also each time the `scroll` or `resize` event fires.